js/app.tsx

```diff
@@ -1,3 +1,4 @@
+import "./sentry.js";
```

Comment:
Right now all the side effects happen during the import because they're top level in `sentry.ts`, which makes it hard to track all the things that happen during initialization.

Could you wrap all the code in that file in a `export const init = () => {}`, and then call the init from this file?

(Also, this is written `./sentry.js`, when I think `./sentry` should be fine. I'm surprised it worked and also surprised eslint didn't catch that.)

Reply:
See my comment here: https://github.com/mbta/orbit/pull/58#discussion_r1691718215

Reply:
whaaat

An alternative could be to add a comment and add an empty line to separate it from all the well-behaved imports:

```suggestion
// runs initialization during import
import "./sentry";

```